cmake/external.cmake

```diff
@@ -150,6 +150,16 @@ function(link_libopenh264 target_name)
     endif()
 endfunction()
 
+# libSRTP
+function(link_libsrtp target_name)
+    find_package(PkgConfig REQUIRED)
+    pkg_check_modules(SRTP libsrtp2)
+    if (SRTP_FOUND)
+        target_include_directories(libsrtp INTERFACE ${SRTP_INCLUDE_DIRS})
+        target_link_libraries(libsrtp INTERFACE ${SRTP_LINK_LIBRARIES})
+    endif()
+endfunction()
```

Comment:
No, it's still wrong, please look at the last revision of the suggestion

Reply:
Yes, I keep updating it.